src/GeneralComponents/GeneralWindow/GeneralWindow.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { WindowContainer, TitleContainer, CloseButton, ContentContainer } from './styles';
+
+export default function GeneralWindow({ children, unmountGeneralWindow, rightSided, title, titleId, id, containerId }) {
+	let position;
+	if (rightSided) {
+		position = 'rightSided';
+	} else position = 'centered';
+
```

Comment:
changed bool checking to matching strings where `center` is default.
also removed unnecessary props `titleId`, `id`, and `containerId`.